src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -25,12 +37,29 @@ export interface ConversationProps {
 
 export const PAGE_SIZE: number = 15
 
+const StyledBox = styled(Flex)`
+  border-top: 1px solid ${color("black10")};
+  padding: 15px 20px;
```

Comment:
We should always stay within our theme spacing scale, and use props on the component (`p={2}`, for example -- this component can likely go away and `<Flex>` styled directly)

Reply:
This makes sense to me

Reply:
Is there a way to make sure this is consistent with the designs we get from designers. Like are they aware of this (i.e. 15px not being an option in our theme and to not use it in specs)?

Reply:
Other than reporting back to the designer directly and helping to educate, not too sure. We, in engineering, are the last guard against design drift! So if some design is "15px 20px", and we know that there's a `1 = 10px, and 2 -20px` in our design system theme, then we can either round that `15px` up or down using our best judgement / eye. But we never step out of the design system spacing scale which only creates further problems at scale. Most likely, the designer just isn't being pixel perfect here (which is fine since engineers can stick within the theme and values "snap" to their closest targets.)